node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -80,6 +81,7 @@ const (
 	VolumeStorageIdsDelimiter   = ";"
 	WwnOuiEnd                   = 7
 	WwnVendorIdentifierEnd      = 16
+	flushBufsFlag               = "--flushbufs"
```

Comment:
I would put it right after `blockDevCmd`

Reply:
done

Reply:
done